mmagic/apis/mmagic_inferencer.py

```diff
@@ -109,6 +109,7 @@ class MMagicInferencer:
         # text2image models
         'controlnet',
         'disco_diffusion',
+        'fastcomposer',
```

Comment:
If you add fastcomposer to text2image inferencer, you should make sure that fastcomposer could run with mmagic/apis/inferencers/text2image_inferencer.py. If not, you would delete this line.